README.md

```diff
@@ -40,10 +40,13 @@ logic efficiently.
 
 The above objectives can only be achieved by building on a solid mathematical
 foundation.  The formal model that underpins our system, also called DBSP, is
-described in the accompanying paper:
+described in the accompanying paper (link points to an earlier version while
```

Comment:
I suggest we point the reference to this version, which is much better than the one on archive and is very close to final. Just add  a [preprint] label to the PDF.

Reply:
Amazing how many mistakes one can do in a small edit.
Yes, but I don't want to commit the pdf yet. Once we have a camera-ready we'll switch.
I am secretly hoping they will give us extra space so we can slip one more example.
If not, we'll have to do some work to squeeze this version to fit the extra paragraphs I added.

Reply:
Your call, but we keep hearing that people tried to read the arxiv paper, and didn't fully understand it.  I don't see any downside to pointing them to a much better write-up even if it's not final.

Reply:
For papers having multiple versions is not as good. Let's just wait a few days until we get instructions for the camera-ready and we'll package a final version. We could perhaps create a longer version with additional examples and less math if you want that.